x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/update_rules.ts

```diff
@@ -135,10 +137,40 @@ export const updateRules = async ({
     }
   );
 
+  let ruleCurrentStatus;
+  if (savedObjectsClient != null) {
+    ruleCurrentStatus = await savedObjectsClient.find<IRuleSavedAttributesSavedObjectAttributes>({
+      type: ruleStatusSavedObjectType,
+      perPage: 1,
+      sortField: 'statusDate',
+      sortOrder: 'desc',
+      search: rule.id,
+      searchFields: ['alertId'],
+    });
+  }
```

Comment:
nit:

```ts
  const ruleCurrentStatus =
    savedObjectsClient ??
    (await savedObjectsClient.find<IRuleSavedAttributesSavedObjectAttributes>({
      type: ruleStatusSavedObjectType,
      perPage: 1,
      sortField: 'statusDate',
      sortOrder: 'desc',
      search: rule.id,
      searchFields: ['alertId'],
    }));
```

This will remove the `let`

Reply:
ahh nice thank you. I'll update that.

Reply:
Per conversations:

Make it mandatory in TypeScript and then add it everywhere else.